radis/lbl/loader.py

```diff
@@ -496,6 +497,44 @@ def __init__(self):
         self.total_lines = 0  #: int : number of lines in database.
 
 
+class CalcTime(ConditionDict):
+    """A class to hold Spectrum calculation time dependent parameters, under the attribute
+    :py:attr:`~radis.lbl.loader.DatabankLoader.time.dict_time` of
+    :py:class:`~radis.lbl.factory.SpectrumFactory`.
+
+    It also stores functions to print all the entities based on verbose value
+
+    See Also
+    --------
+
+    :py:attr:`~radis.lbl.loader.DatabankLoader.input`,
+    :py:attr:`~radis.lbl.loader.DatabankLoader.time`,
+
+    """
+
+    def __init__(self, verbose):
+        super(CalcTime, self).__init__()
+
+        # Dev: Init here to be found by autocomplete
+        self.initial = {}
+        self.dict_time = {}
+        self.verbose = verbose
+
+    def start(self, key, verbose):
+        if __debug__:
+            self.initial[key] = {"start_time": time(), "verbose": verbose}
+
+    def stop(self, key, details=""):
```

Comment:
Adding argument `details` here because there are many instances where we  need a particular value to print like `jit` which is declared after the `start` point and also makes easier to print statements in `if elif else` type condition by having a common starting point!